eclipse-che/che-operator
# Gh19707 secure workspace

### What does this PR do?
Secures workspace route so only owner has access to it.


### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/19707

### How to test this PR?

che-operator image `quay.io/mvala/che-operator:gh19707-secureWorkspace`

1. deploy on openshift with devworkspaces enabled 
2. login as some user and start a devworkspace
3. copy workspace url or get it from devworkspace object `kubectl get dw -n user1-che`
4. login as different user and try to access the workspace, you should get Unauthorized 403.


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
